ui/projekt-monsterz-front/src/app/login-screen/login-screen.component.css

```diff
@@ -0,0 +1,31 @@
+* {
+     padding: 5px;
+     margin: 0;
+     
+}
+div:first-of-type {
```

Comment:
Interesting identifier to use, as opposed to a class or something. Any particular reason?